docs/zh_cn/user_guides/inference.md

```diff
@@ -1,87 +1,96 @@
+# 从 checkpoint 中加载 Dataset_meta，并将其传递给模型的 init_detector
+visualizer.dataset_meta = model.dataset_meta
+
 # 测试单张图片并展示结果
 img = 'test.jpg'  # 或者 img = mmcv.imread(img)，这样图片仅会被读一次
 result = inference_detector(model, img)
-# 在一个新的窗口中将结果可视化
-model.show_result(img, result)
-# 或者将可视化结果保存为图片
-model.show_result(img, result, out_file='result.jpg')
-
-# 测试视频并展示结果
-video = mmcv.VideoReader('video.mp4')
-for frame in video:
-    result = inference_detector(model, frame)
-    model.show_result(frame, result, wait_time=1)
-```
 
-jupyter notebook 上的演示样例在 [demo/inference_demo.ipynb](https://github.com/open-mmlab/mmdetection/blob/master/demo/inference_demo.ipynb) 。
+# 显示结果
+img = mmcv.imread(img)
+img = mmcv.imconvert(img, 'bgr', 'rgb')
 
-## 异步接口-支持 Python 3.7+
 
-对于 Python 3.7+，MMDetection 也有异步接口。利用 CUDA 流，绑定 GPU 的推理代码不会阻塞 CPU，从而使得 CPU/GPU 在单线程应用中能达到更高的利用率。在推理流程中，不同数据样本的推理和不同模型的推理都能并发地运行。
+visualizer.add_datasample(
+    'result',
+    img,
+    data_sample=result,
+    draw_gt=False,
+    show=True)
 
-您可以参考 `tests/async_benchmark.py` 来对比同步接口和异步接口的运行速度。
-
-```python
-import asyncio
-import torch
-from mmdet.apis import init_detector, async_inference_detector
-from mmdet.utils.contextmanagers import concurrent
-
-async def main():
-    config_file = 'configs/faster_rcnn/faster_rcnn_r50_fpn_1x_coco.py'
-    checkpoint_file = 'checkpoints/faster_rcnn_r50_fpn_1x_coco_20200130-047c8118.pth'
-    device = 'cuda:0'
-    model = init_detector(config_file, checkpoint=checkpoint_file, device=device)
+# 测试视频并展示结果
+# 构建测试 pipeline
+model.cfg.test_dataloader.dataset.pipeline[0].type = 'LoadImageFromNDArray'
+test_pipeline = Compose(model.cfg.test_dataloader.dataset.pipeline)
 
-    # 此队列用于并行推理多张图像
-    streamqueue = asyncio.Queue()
-    # 队列大小定义了并行的数量
-    streamqueue_size = 3
+# 初始化可视化工具
+visualizer = VISUALIZERS.build(model.cfg.visualizer)
+# 从 checkpoint 中加载 Dataset_meta，并将其传递给模型的 init_detector
+visualizer.dataset_meta = model.dataset_meta
 
-    for _ in range(streamqueue_size):
-        streamqueue.put_nowait(torch.cuda.Stream(device=device))
+# 显示间隔 (ms), 0 表示暂停
+wait_time = 1
 
-    # 测试单张图片并展示结果
-    img = 'test.jpg'  # or 或者 img = mmcv.imread(img)，这样图片仅会被读一次
+video = mmcv.VideoReader('video.mp4')
 
-    async with concurrent(streamqueue):
-        result = await async_inference_detector(model, img)
+cv2.namedWindow('video', 0)
 
-    # 在一个新的窗口中将结果可视化
-    model.show_result(img, result)
-    # 或者将可视化结果保存为图片
-    model.show_result(img, result, out_file='result.jpg')
+for frame in track_iter_progress(video_reader):
+    result = inference_detector(model, frame, test_pipeline=test_pipeline)
+    visualizer.add_datasample(
+        name='video',
+        image=frame,
+        data_sample=result,
+        draw_gt=False,
+        show=False)
+    frame = visualizer.get_image()
+    mmcv.imshow(frame, 'video', wait_time)
 
+cv2.destroyAllWindows()
+```
 
-asyncio.run(main())
+jupyter notebook 上的演示样例在 [demo/inference_demo.ipynb](../../../demo/inference_demo.ipynb) 。
```

Comment:
```suggestion
jupyter notebook 上的演示样例在 [demo/inference_demo.ipynb](https://github.com/open-mmlab/mmdetection/blob/3.x/demo/inference_demo.ipynb) 。
```
![image](https://user-images.githubusercontent.com/27466624/217768770-94f6c234-d34d-401d-bf34-f47562b11ee3.png)
Do not use relative links except for `.md` links. Otherwise readthedocs won't render correctly.

Reply:
Thanks for your suggestions. And four relative paths in this file have been changed to static links.

But one thing that needs to be mentioned here is: 
Relative links are used in the corresponding English version doc.

Thanks again.

Reply:
![image](https://user-images.githubusercontent.com/27466624/219524546-308c0764-83c6-42c0-ba23-a89e42d0c9a9.png)
@DanShouzhu, Relative links in the corresponding English doc should also be modified.